.github/workflows/semantic.yml

```diff
@@ -0,0 +1,20 @@
+name: "Check Semantic Commit"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - name: semantic-pull-request
+        uses: amannn/action-semantic-pull-request@v4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          validateSingleCommit: true
```

Comment:
Our current behaviour for this is `false`: https://github.com/electron/electron/blob/fec147a0cbff493a7ca1579a3eb0351c2c52cc7f/.github/semantic.yml#L2

Reply:
hm... okay. I'll switch it to `false` but I think we may actually want this to be `true` because we have merged a bunch of non-semantic commits by accident. e.g. https://github.com/electron/electron/commit/92c5dedc76275fb84814aa05ab71c85d84a05663

Reply:
That's true. The trade-off is that folks are used to editing PR titles to make sure things are semantic and it annoyingly doesn't work with single-commit PRs.

Would be nice if `squash and merge` always defaulted to PR title rather than commit message regardless of the number of commits in a PR, but that's for the :octocat: product team to figure out.